docs/src/components/components/activity-log/activity-log.html

```diff
@@ -0,0 +1,40 @@
+<ul class="activity_log">
```

Comment:
I think the activity log should belong in `features` because in order to render correctly, it needs to contain other components like `elastic_line` but I'm open to it belonging here too.